lib/challenge.js

```diff
@@ -48,7 +66,14 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
+  name: "Spud",
+  species: "Dog",
+  breed: "Lab",
+  noise: = function () {
```

Comment:
That equal sign is a syntax error in an object literal. However, if you simply remove just the equal sign, this is exactly what the question was asking for.